com.unity.render-pipelines.core/Editor/Volume/VolumeProfileEditor.cs

```diff
@@ -23,6 +23,11 @@ public override void OnInspectorGUI()
         {
             serializedObject.Update();
             m_ComponentList.OnGUI();
+
+            EditorGUILayout.Space();
+            if (m_ComponentList.hasHiddenVolumeComponents)
+                EditorGUILayout.HelpBox("There are Volume Components that are hidden in this asset because they are incompatible with the current active render pipeline. Change the active Render Pipeline to see them.", MessageType.Info);
```

Comment:
```suggestion
                EditorGUILayout.HelpBox("There are Volume Components that are hidden in this asset because they are incompatible with the current active Render Pipeline. Change the active Render Pipeline to see them.", MessageType.Info);
```